src/wad/level.rs

```diff
@@ -152,12 +150,14 @@ impl Level {
         let mut min_light = sector.light;
         let sector_id = self.sector_id(sector);
         for line in self.linedefs.iter() {
-            let (left, right) =
-                (match self.left_sidedef(line) {
-                    Some(l) => l.sector, None => continue,
-                 }, match self.right_sidedef(line) {
-                    Some(r) => r.sector, None => continue,
-                 });
+            let (left, right) = (match self.left_sidedef(line) {
+                Some(l) => l.sector,
```

Comment:
The indentation here is seriously wrong. It should at least be aligned to the `match`, right? Could you please fix this up?